### PR TITLE
break out shared parts of protobuffer into their own proto files

### PR DESCRIPTION
**Pull request by kevinkreiser**
this makes moving them from one proto object to another very easy

### CODE REVIEW

File: src/odin/directionsbuilder.cc [hunk 1]
```diff
@@ -132,47 +132,7 @@ TripDirections DirectionsBuilder::PopulateTripDirections(
...
-          == TripPath_Location_SideOfStreet_kRight) {
-        direction_location->set_side_of_street(
-            TripDirections_Location_SideOfStreet_kRight);
-      } else {
-        direction_location->set_side_of_street(
-            TripDirections_Location_SideOfStreet_kNone);
-      }
-    }
-  }
+  trip_directions.mutable_location()->CopyFrom(etp->location());
```
Comment (kevinkreiser) [inline src/odin/directionsbuilder.cc]: that stuff above is now one line :smile:

**Comment by kevinkreiser** [comment]
@kdiluca @dgearhart

**Comment by kdiluca** [comment]
:+1:

**Comment by dgearhart** [comment]
:shipit:

**Pull request merged**